PFSD doesnt tell user if they attempted to join incorrect pool

Discovery server gives a good log for  attempting to join incorrect pool

```
2016/03/09 23:43:51 [ERROR] main: Join: node belongs to pool peaceful_rabbit but tried to join pool asd
```

PFSD doesnt tell you what the error is directly

```
2016/03/09 23:43:51 [FATAL] dnetclient: Failure dialing discovery server after multiple attempts, Giving up Could not join the pool
```

PFSD should probably have an error stating what pool it expects you to connect to.

--- a/pfsd/dnetclient/join.go
+++ b/pfsd/dnetclient/join.go
@@ -1,7 +1,7 @@
 package dnetclient
 
 import (
-	"errors"
+	"fmt"
 	"github.com/cpssd/paranoid/pfsd/globals"
 	pb "github.com/cpssd/paranoid/proto/discoverynetwork"
 	"golang.org/x/net/context"
@@ -14,7 +14,7 @@ import (
 func Join(pool string) error {
 	conn, err := dialDiscovery()
 	if err != nil {
-		return errors.New("Failed to dial discovery server")
+		return fmt.Errorf("failed to dial discovery server: %s", err)
 	}
 	defer conn.Close()
 
@@ -31,7 +31,7 @@ func Join(pool string) error {
 			},
 		})
 	if err != nil {
-		return errors.New("Could not join the pool")
+		return fmt.Errorf("could not join the pool: %s", err)
 	}
 
 	interval := response.ResetInterval / 10 * 9

